projects/spectator/src/lib/select-option.ts

```diff
@@ -11,7 +11,7 @@ import { isString } from './types';
  * selectOption('al' | ['al', 'ab'], select, config);
  */
 export function selectOption(
-  options: string | string[],
+  options: string | string[] | HTMLOptionElement | HTMLOptionElement[],
```

Comment:
Create a type an use it in both places, please.

Reply:
I created a type SelectOptions which is in the types.ts file. However when I imported into matchers-types.d.ts it broke the tests (and the build) so I changed it back in the last commit. Let me know if you know how I can import it without breaking the tests.